application-forum-ui/src/main/resources/ForumCode/NewFlagSheet.xml

```diff
@@ -60,26 +60,46 @@
   ## sending email
   #set($emailRegex = "[-0-9a-zA-Z.+_]+@[-0-9a-zA-Z.+_]+\.[a-zA-Z]{2,4}")
   #set($adminEmail = $xwiki.getXWikiPreference('admin_email'))
+  #if ("${adminEmail}" == '')
+    ## Get admin email from global administration settings
+    #set ($configDocName = 'Mail.MailConfig')
+    #set ($configDoc = $xwiki.getDocument($configDocName))
```

Comment:
Since the `$configDocName` is used only once, you can pass it directly as parameter in `$xwiki.getDocument()`.